src/pages/settings/expense-categories/Edit.tsx

```diff
@@ -72,16 +72,17 @@ export function Edit() {
         .catch((error: AxiosError) => {
           toast.dismiss();
 
+          queryClient.invalidateQueries('/api/v1/expense_categories');
+
+          queryClient.invalidateQueries(
+            route('/api/v1/expense_categories/:id', { id })
+          );
```

Comment:
No, it's not enough. I've tested it but the data won't be fetched again if I turn off invalidate query for endpoint with id. In my opinion this is fine if it works like this because actually ``/api/v1/expense_categories'' and ``/api/v1/expense_categories/:id'' are two different endpoints.